tests/test_infer/test_models/test_bloom.py

```diff
@@ -0,0 +1,140 @@
+import os
```

Comment:
Seems to be redundancy to have different files testing different models. Might want to create some mapping (E.g. mapping for configs, tokenizers, CausalLM classes for different models) and use the same script to test different models